authors/apps/profiles/tests/test_follow.py

```diff
@@ -39,7 +44,7 @@ def test_followed_Successfully(self):
             '/api/profiles/Thanos/follow/')
         self.assertEqual(res.status_code, status.HTTP_202_ACCEPTED)
         self.assertEqual(json.loads(res.content), {'profile': {
-            'username': 'Thanos', 'first_name': '', 'last_name': '', 'bio': ''}})
+            'username': 'Thanos', 'first_name': '', 'last_name': '', 'bio': '', 'following': True}})
```

Comment:
Why are you hard coding this?

Reply:
@graycadeau let me fix that

Reply:
@graycadeau I have removed the hard coding